assets/src/components/iconAlertCircle.tsx

```diff
@@ -1,20 +1,41 @@
 import React from "react"
 
-const IconAlertCircle = () => (
+export enum AlertIconStyle {
+  Black = 1,
+  Grey,
+  Highlighted,
+}
+
+const styleClass = (style: AlertIconStyle): string => {
+  switch (style) {
+    case AlertIconStyle.Black:
+      return "--black"
+    case AlertIconStyle.Grey:
+      return "--grey"
+    case AlertIconStyle.Highlighted:
+      return "--highlighted"
+  }
+}
+
+export interface Props {
+  style: AlertIconStyle
+}
+
+const IconAlertCircle = (props: Props) => (
   <svg viewBox="-3 -3 54 54" className="c-icon-alert-circle">
-    <IconAlertCircleSvgNode />
+    <IconAlertCircleSvgNode {...props} />
   </svg>
 )
 
-export const IconAlertCircleSvgNode = () => (
-  <>
+export const IconAlertCircleSvgNode = ({ style }: Props) => (
```

Comment:
Is there a reason to consider having a default style?

Reply:
I don't think there is yet.